merge all the config objects

ttung opened the pull request:
### Description
Now with #115, there's no need to have separate config objects.  Collapse it all into one.

Depends on #115 obviously.

### Test plan
1. pytest
1. flask run

shanaxel42 reviewed (approved):
ONE CONFIG TO RULE THEM ALL

Commit:
merge all the config objects

### Description
Now with #115, there's no need to have separate config objects.  Collapse it all into one.

### Test plan
1. pytest
1. flask run
